perf/perf.md

```diff
@@ -0,0 +1,124 @@
+# Perf <!-- omit in toc -->
+
+| Lifecycle Stage | Maturity      | Status | Latest Revision |
+| --------------- | ------------- | ------ | --------------- |
+| 1A              | Working Draft | Active | r0, 2022-11-16  |
+
+Authors: [@marcopolo]
+
+Interest Group: [@marcopolo], [@mxinden], [@marten-seemann]
+
+[@marcopolo]: https://github.com/mxinden
+[@mxinden]: https://github.com/mxinden
+[@marten-seemann]: https://github.com/marten-seemann
+
+# Table of Contents <!-- omit in toc -->
+- [Context](#context)
+- [Protocol](#protocol)
+- [Benchmarks](#benchmarks)
+  - [Single connection throughput](#single-connection-throughput)
+  - [Handshakes per second](#handshakes-per-second)
+- [Security Considerations](#security-considerations)
+- [Prior Art](#prior-art)
+
+# Context
```

Comment:
```suggestion
## Context
```
 Nit: Multiple L1 headings in this document.

Reply:
Why is that a problem? This is a top level section. If we made this `##` then it would be a child of ToC, which is wrong.

Reply:
Ah I see the argument now. This could all be a child of the `# Perf` part. I think that's a good argument, but the counter argument is that then you would always lose one nesting level in this style since you'll always have that title heading.

Unless we have some specific templating system we are trying to make happy, I don't think it's necessary to only have a single `#` level. This whole doc is about `perf`, so it should be okay to have multiple `#` levels.